Fix endless episode details scrolling

SergioEstevao opened the pull request:
| 📘 Part of: # |  <!-- project issue number, if applicable -->
|:---:|

Fixes #1997 
Fixes #2626 <!-- issue number, if applicable -->

<!-- Please include a summary of what this PR is changing and why these changes are needed. -->
Ensures that web view as correct size and HTML is fully loaded before calculating scroll size.

## To test

1. Start the app
2. Play an episode with long episode details
3. Open the full screen player by dragging up slowly the mini-player
4. Tap on the details tab
5. Scroll down and see if you don't see a large empty space on the details text at the bottom
6. Scroll up again and check if you can dismiss the player by scrolling all way up and continue

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

SergioEstevao committed:
Ensure that web view as a size and HTML is completely loaded before calculating a size

dangermattic commented:
<!--
  0 Errors
  1 Warning: This PR is assigned to the mil...
  0 Messages
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/Automattic/pocket-casts-ios/milestone/72">7.80 ❄️</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

marlonjames71 commented:
I can reproduce the every time so far. Tapping on the mini player has correct content size in the scroll view. But doing a longer swipe up on the mini player causes the incorrect content size.

SergioEstevao committed:
Only ensure that the view as proper width and minimum height so height calculations are correct.

If the view was set to larger height the javascript method will always return as a minimum the view height

SergioEstevao committed:
Ensures that webview always a valid frame and no scroll is needed

SergioEstevao committed:
Ensure scroll size is update when is displayed

SergioEstevao committed:
Fix lint issues

danielebogo reviewed (approved):
Thanks @SergioEstevao ! It works!
Tested with long and short episode descriptions, with tap and drag gestures. I don't see the infinite content size anymore. :shipit:

danielebogo committed:
Update Chnagelog
